Assets/Events/Threat.json

```
@@ -0,0 +1,174 @@
+    "image": "behold",
+    "type": "Threat",
+    "outcomes": [
+      {
+        "type": "FlavourText",
+        "customDescription": "Your townsfolk start having the strangest dreams..."
+      },
+      {
+        "type": "ThreatAdded",
+        "baseAmount": 4
+      }
+    ]
+  },
+  {
+    "name": "time-being",
+    "headline": "Tributes to the Time Being",
+    "article": "A local cult has started making offerings to a mysterious creature, in exchange for future sight. \"We have a busy schedule ahead, so let's just leave this for the Time Being\" chant the cultists.",
+    "image": "empty-hourglass",
+    "type": "Threat",
+    "outcomes": [
+      {
+        "type": "FlavourText",
+        "customDescription": "Your townsfolk are fearful of this 'Time Being'"
+      },
+      {
+        "type": "ThreatAdded",
+        "baseAmount": 3
+      }
+    ]
+  },
+  {
+    "name": "beetles",
+    "headline": "Hello Beetles, Goodbye Beetles",
+    "article": "Giant monstrous beetles migrating from the strawberry fields terrorising the town this morning with loud melodic buzzing from the rooftops. Town etymologist had hoped they would have moved along yesterday. The blackbird population is reportedly under attack from these creatures. Local blacksmith Maxwell claims he has just the right tool for the situation...",
+    "image": "scarab-beetle",
+    "type": "Threat",
+    "outcomes": [
+      {
+        "type": "FlavourText",
+        "customDescription": "We don't know how much longer we can let it be.."
+      },
+      {
+        "type": "ThreatAdded",
+        "baseAmount": 3
+      }
+    ]
+  },
+  {
+    "name": "feeding-day",
+    "headline": "Feeding Day is Upon Us!",
+    "article": "As it does every year, town prepares for its annual feeding day, when townsfolk come together to celebrate the sating of the beast. As is tradition, each home leaves a fresh ham upon their doorstep tomorrow night. Residents are being urged to purchase their hams quickly to avoid a long and agonising death for them and their loved ones this holiday season.",
+    "image": "behold",
+    "type": "Threat",
+    "outcomes": [
+      {
+        "type": "FlavourText",
+        "customDescription": "The beast will be fed for another year"
+      },
+      {
+        "type": "ThreatAdded",
+        "baseAmount": 5
+      }
+    ]
+  },
+  {
+    "name": "humming-lights",
+    "headline": "Mysterious Lights won't Stop Humming",
+    "article": "The strange dancing lights off the east side of town have started singing a little tune! Some claim it's cheerful, others ominous. Local residents have reported asking, \"Oh God, oh God why won't they stop? Please, someone, make them stop.\"",
+    "image": "behold",
+    "type": "Threat",
+    "outcomes": [
+      {
+        "type": "FlavourText",
+        "customDescription": "The purpose of these lights are still to be determined"
```

```suggestion
        "customDescription": "The purpose of these lights is still to be determined"
```
I think the subject is "the purpose", singular